packages/vite/src/node/plugins/css.ts

```diff
@@ -1001,9 +1002,14 @@ function loadPreprocessor(lang: PreprocessLang, root: string): any {
     const resolved = require.resolve(lang, { paths: [root, ...fallbackPaths] })
     return (loadedPreprocessors[lang] = require(resolved))
   } catch (e) {
-    throw new Error(
-      `Preprocessor dependency "${lang}" not found. Did you install it?`
-    )
+    if (isDebug) {
+      debug(`[load] Preprocessor dependency "${lang}" not found. Did you install it?`);
+      throw new Error(e.stack);
+    } else { 
+      throw new Error(
+        `Preprocessor dependency "${lang}" not found. Did you install it?`
+      )
+    }
```

Comment:
I think the error cause is useful even when you aren't debugging here. If there is a way to identify when the problem is that it wasn't installed, issue this message (that is friendly for 99% of the cases) and if it is installed, showing something like `Preprocessor dependency "{lang} failed to load. ${e.message}`
But we can leave it for a future PR, this is already an improvement. If the above isn't easy to implement, maybe we should move to a variable the repeated text before merging

Reply:
I think this should do it